src/components/SliderBox/slidesData.ts

```diff
@@ -0,0 +1,19 @@
+interface SlidesArrayType {
```

Comment:
create a mocks folder inside SliderBox and move this file here

Reply:
fixed